### PR TITLE
use fixed version of proto instead of master branch

### PR DESCRIPTION
**Pull request by marsishandsome**
raft-rs master branch has some error.
change master branch to fixed version.

image

### CODE REVIEW

File: tikv-client/scripts/proto.sh [hunk 1]
```diff
@@ -16,20 +16,25 @@
...
-	git clone https://github.com/pingcap/raft-rs
+	git clone https://github.com/pingcap/raft-rs; cd raft-rs; git checkout ${raft_rs_hash}; cd ..
 fi
 
 if [ -d "tipb" ]; then
-	cd tipb; git pull origin master; cd ..
+	cd tipb; git fetch -p; git checkout ${tipb_hash}; cd ..
 else
 	git clone https://github.com/pingcap/tipb
+	git clone https://github.com/pingcap/tipb; cd tipb; git checkout ${tipb_hash}; cd ..
```
Comment (birdstorm) [inline tikv-client/scripts/proto.sh]: DUPLICATE `git clone https://github.com/pingcap/tipb`

Comment (marsishandsome) [reply tikv-client/scripts/proto.sh]: fixed

**Comment by marsishandsome** [comment]
/run-all-tests tikv=release-2.1 tidb=release-2.1 pd=release-2.1

**Comment by zyguan** [comment]
/run-all-tests tikv=release-2.1 tidb=release-2.1 pd=release-2.1

**Pull request merged**